tests/testthat/test-module_teal.R

```diff
@@ -65,6 +73,178 @@ transform_list <<- list(
   )
 )
 
+
+# decorators ------------------------------------------------------------------------------------------------------
+
+
+gg_xlab_decorator <<- function(output_name) {
```

Comment:
this is unnecessary complicated decorator. `ggplot2` shouldn't be in depedencies of `teal`. This test can use some more generic decorators - like one you did in example_module

Reply:
`ggplot2` is in dependencies of `teal.widgets` https://github.com/insightsengineering/teal.widgets/blob/main/DESCRIPTION#L27

But agree, we can reuse `example_module()` - the same as as we did on `shinytest2` - which are simpler